### ISSUE TITLE
Injected code goes in the wrong chunk

### ISSUE THREAD

**Issue by alok-runner** [marco-prontera/vite-plugin-css-injected-by-js#58]
Does not appear to be working with `vite` v4 (and `@vitejs/plugin-react` v3 if that matters)

It seems to silently fail and not include the css anywhere.

**Comment by emhamzahazeen** [comment]
I second that. Css code is not present in any of the generated files.

**Comment by marco-prontera** [comment]
I tried with Vite v4 and it works. It works also with `@vitejs/plugin-react` v3. I think both of you should give me more information about your Vite configurations

**Comment by luesoad** [comment]
I'm using @sveltejs/vite-plugin-svelte and I'm having the same problem
```
import { defineConfig } from "vite";
import { svelte } from "@sveltejs/vite-plugin-svelte";
import postcss from "./postcss.config";
import { resolve } from "path";
import cssInjectedByJsPlugin from "vite-plugin-css-injected-by-js"; // 

export default defineConfig({
  plugins: [svelte({ emitCss: false }), cssInjectedByJsPlugin()],
  css: { postcss },
  build: {
    lib: {
      entry: resolve(__dirname, "src/main.ts"),
      name: "test",
      // the proper extensions will be added
      fileName: "test",
    },
    rollupOptions: {
      external: ["svelte"],
      output: {
        globals: {
          svelte: "Svelte",
        },
      },
    },
  },
});
```

**Comment by dvnrsn** [comment]
Thanks for your work on this plugin! I've been looking for a solution on this for a few days. I hope I can get it to work.

I'm using vite 4.0.3, rollup, tailwind with a postcss config that is pretty generic

```js
/// postcss.config.js
module.exports = {
  plugins: {
    tailwindcss: {},
    autoprefixer: {},
  },
}
```
I don't believe the `css: { postcss },` @luesoad has is necessary as doesn't Vite automatically grab postcss config?

I do see that there is no outputted CSS file but I am not seeing any imports in the JS and there are no Network calls for css.

```js
// vite.config.ts
export default defineConfig({
  base: '/'
  plugins: [cssInjectedByJsPlugin()],
  build: {
    rollupOptions: {
      input: path.resolve(__dirname, '_main.js'),
    },
  },

**Comment by dvnrsn** [comment]
Sure here's a repo @marco-prontera https://github.com/dvnrsn/vite-postcss.

When I run pnpm build, I get a bundle that doesn't include an import afaict.

**Comment by marco-prontera** [comment]
fixed in [v2.3.0](https://github.com/marco-prontera/vite-plugin-css-injected-by-js/releases/tag/v2.3.0)
